blog/docs/articles/configurable-runtimeclassnames.md

```diff
@@ -0,0 +1,67 @@
+# Configurable RuntimeClassNames
+
+**Author: Caleb Woodbine**
+
+Starting in Knative Serving v1.15, administrators are now able to configure the default the RuntimeClassName field for deployments as default and via a Knative Service label selector.
+
+## Runtime Classes
+
+**What is a Runtime Class?**
+
+Runtime Classes configure the runtime of a container with settings and handlers, such as `runc`, `crun`, `runsc`, `nvidia`/`nvidia-cdi` or `kata`. `runc` being the defacto which provides kernel-level isolation which is often unsuitable for running untrusted workloads.
```

Comment:
Could you expand a little bit on what you mean by settings and handlers here?

Reply:
I've updated it now. What do you think?